sui/ibc/sources/bcs_utils.move

```diff
@@ -172,4 +172,36 @@ module ibc::bcs_utils {
     //     };
```

Comment:
bad news, this file is taken from aptos probably and these utils are already there in SUI std. Please check if this file is used at all, if not let's remove it entirely.